browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -3107,6 +3107,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			control.style.gridColumn = 'span ' + parseInt(data.width);
 		}
 
+		if(control && (data.id === 'viewgrid' || data.id === 'favgrid')) {
```

Comment:
please do not hardcode rules based on widget id

Reply:
can't we do that in CSS?

Reply:
I’ve removed the hardcoded rules based on the widget Id and moved the relevant styling into the CSS file.